File: _src/07/Chapter7.VB.Data/Context.vb
Imports System.Data.Entity
Imports Chapter7.VB.Data.Models
Imports Chapter7.VB.Data.Maps

Public Class Context
    Inherits DbContext
    Public Sub New()
        MyBase.New("name=chapter7")
    End Sub
    Property People As DbSet(Of Person)
    Property Companies() As DbSet(Of Company)

    Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
        MyBase.OnModelCreating(modelBuilder)
        modelBuilder.Configurations.Add(New PersonMap())
    End Sub
End Class

File: _src/02/Chapter2.CSharp/Context.cs
using System.Data.Entity;

namespace Chapter2.CSharp
{
    public class Context : DbContext
    {
        public Context()
            : base("name=chapter2")
        {

        }
        public DbSet<Person> People { get; set; }
        public DbSet<Company> Companies { get; set; }
    }
}
